scrc/preprocessors/extractors/spider_specific/section_splitting_functions.py

```diff
@@ -25,13 +27,104 @@ def XX_SPIDER(decision: Union[bs4.BeautifulSoup, str], namespace: dict) -> Optio
     pass
 
 
+def VD_Omni(decision: Union[bs4.BeautifulSoup, str], namespace: dict) -> Optional[Dict[Section, List[str]]]:
```

Comment:
Is there a reason why you don't use the associate_sections function? I think it would be easiest for you to just define the get_paragraphs() function and the regexes in all_section_markers.

Reply:
Thanks for your question. If it is ok to add a Section. Composition to the  all_section_markers, sure, I will do it with pleasure. Otherwise, I feel a bit lost.  As my task is not section splitting, I am not sure what to put for Section.{FACTS, CONSIDERATIONS, RULINGS, FOOTER}. Is it ok to use the values that are already declared for the Language.FR:

 Language.FR: {
            Section.FACTS: [r'Faits\s?:', r'en fait et en droit', r'(?:V|v)u\s?:', r'A.-'],
            Section.CONSIDERATIONS: [r'Considérant en (?:fait et en )?droit\s?:', r'(?:C|c)onsidérant(s?)\s?:',
                                     r'considère'],
            Section.RULINGS: [r'prononce\s?:', r'Par ces? motifs?\s?', r'ordonne\s?:'],
            Section.FOOTER: [
                r'\w*,\s(le\s?)?((\d?\d)|\d\s?(er|re|e)|premier|première|deuxième|troisième)\s?(?:janv|févr|mars|avr|mai|juin|juill|août|sept|oct|nov|déc).{0,10}\d?\d?\d\d\s?(.{0,5}[A-Z]{3}|(?!.{2})|[\.])',
                r'Au nom de la Cour'
            ]
        }

Reply:
Yes exactly. I would just go from the values that are already defined, and see if it works already out of the box for your spider. Maybe you just have to add a few more terms and you are ready to go.

Reply:
Unfortunately, it does not work out of the box (not even for a single decision). I would suggest that we put the full-fledged section splitting of French cantons for the future because editing these rules is yet another project and I regret that this semester I am not able to work on two projects simultaneously.

Another issue is that in court_composition_extractio_functions, input is defined to be the header, while in some spiders (such as GE_Gerichte), the composition is located at the end of the document. Therefore, I believe there are some inconsistencies between the section splitting and the court composition extraction tasks.